templates/taxi/car_detail.html

```diff
@@ -3,20 +3,35 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
+  {% if user.is_authenticated %}
+      {% if user not in car.drivers.all %}
+        <form action= "{% url 'taxi:add-driver' pk=car.id %}" method="post">
```

Comment:
This can be one form with different inputs and different values that you will later compare in your view. Also you can include a hidden field here with car_id so you can pull it out in post method in your car detail view